config/settings.yml

```diff
@@ -19,3 +19,13 @@ plugins:
     default: false
     client: true
   assign_mailer_enabled: false
+  remind_assigns:
+    client: true
+    default: 'monthly'
+    type: enum
+    choices:
```

Comment:
I think we should extract the choices into a class under `EnumSiteSetitng` so that the values can be accessed without having to duplicate the choices in multiple places.